### DISCUSSION TITLE
Question: Is it possible to run older Mac OS version VM on an intel mac using UTM?

### DISCUSSION THREAD

**Discussion by stephane-archer** [utmapp/UTM#5863] [UTM Help (macOS)]
<img width="457" alt="Screenshot 2023-11-05 at 10 50 57" src="https://github.com/utmapp/UTM/assets/2981437/1c1d71db-44bd-4d39-8e6f-8b0579e12873">

**Comment by F1248** [comment]
This should also work on Intel Macs, even many times faster than on Apple Silicon. However, it isn't a real virtual Mac but a Hackintosh.
https://khronokernel.github.io/apple/silicon/2021/01/17/QEMU-AS.html

You can find helpful UTM configurations for Mac OS 9 to macOS Monterey 12 here:
https://github.com/adespoton/utmconfigs
